newsarticles/models.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import unicode_literals
+from django.utils.encoding import python_2_unicode_compatible
+from itertools import groupby
+from collections import OrderedDict
+
+from django.db import models
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.auth.models import User
+
+
+@python_2_unicode_compatible
+class NewsSource(models.Model):
+    name = models.CharField(max_length=256)
+    short_name = models.CharField(max_length=256, db_index=True)
+    legacy_feed_id = models.CharField(max_length=8, blank=True, db_index=True)
```

Comment:
You can safely remove `legacy_feed_id`; it was a holdover from before we had a model for `NewsSource`.